packages/AlgernonTests.package/ALGTestCache.class/instance/testCacheKeyGeneration.st

```diff
@@ -1,6 +1,5 @@
 testing
 testCacheKeyGeneration
```

Comment:
isn't there a way to avoid the code duplication with `testCacheKeyGenerationLevel0` ?

Reply:
I think it's not really worth it as still many parameters would need to be passed and the there are not enough duplications.